### PR TITLE
fix: minor UI bugs in provisioning and node modal

### LINKED ISSUE #6890: Add resource fit and finish
Frame 61661

- Learn more link: https://aka.ms/composer-publish-bot#create-new-azure-resources
- Q&A maker icon is missing in the list https://iconcloud.design/search/filter/Azure%20Icons/all/Q&A%20maker/e31aa438e-de64dd68d

### LINKED ISSUE #6970: Installed Node / Still seeing "Node JS Required"
## Describe the bug
1. Create a Bot
2. "Node Required"
3. Install Node. 
4. "Node Required" <--- Unexpected

After installing Node, I've got to exit/restart Composer to pickup the fact that Node is installed. 

## Version
Version:  1.4.0-nightly.235171.cfc0c11
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser
- [X] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS
- [ ] macOS
- [X] Windows
- [ ] Ubuntu

## To Reproduce

1. Fresh install. 
2. Create a Bot
3. Get the "Install Node" Dialog
4. Install Node
5. Create a Bot
6. Get the "Install Node" Dialog  <-- Unexpected. Node is installed. See screen shot. 

## Expected behavior
Create a Bot should work. Node is installed. See screen shot. 

Note: If I Exit/Restart Composer things work. 

## Screenshots

image

### PR DESCRIPTION
**Pull request by pavolum**
## Description

- Making design changes per design spec for publishing modal
- Adding text to node modal indicating need to restart after installation
- Updating template sort order list to reflect new package names coming in Friday
  - More info [here](https://github.com/microsoft/BotFramework-Composer/issues/6978)

## Task Item

fixes #6890 
fixes #6970

### CODE REVIEW

File: extensions/azurePublish/src/components/ChooseProvisionAction.tsx [hunk 1]
```diff
@@ -139,28 +139,28 @@ const ImportActionContent = () => {
       </Summary>
       <Details>
         <ResourceTitle>
-          <Text>{formatMessage('Microsoft Application Registration')}</Text>
+          <Text>{formatMessage('- Microsoft Application Registration')}</Text>
```
Comment (tdurnford) [inline extensions/azurePublish/src/components/ChooseProvisionAction.tsx:142]: Does it make more sense to have the '-' outside of the `formatMessage` function call?
```suggestion
          <Text>- {formatMessage('Microsoft Application Registration')}</Text>
```

File: extensions/azurePublish/src/components/ChooseResourcesList.tsx [hunk 2]
```diff
@@ -47,18 +47,18 @@ const ImageIconPlacholder = styled.div`
...
 const ItemText = styled(Text)`
   font-size: ${FluentTheme.fonts.mediumPlus.fontSize};
-  margin-left: 4px !important;
+  margin-left: 8px !important;
 `;
 
 const ItemTier = styled(Text)`
   font-size: ${FluentTheme.fonts.small.fontSize};
-  margin: 4px 0 0 22px;
+  margin: 4px 0 3px 24px;
```
Comment (tdurnford) [inline extensions/azurePublish/src/components/ChooseResourcesList.tsx:55]: We should try to use multiples of 4px.
```suggestion
  margin: 4px 0 4px 24px;
```

File: extensions/azurePublish/src/components/ChooseProvisionAction.tsx [hunk 3]
```diff
@@ -62,7 +62,7 @@ const InstructionDetails = styled.div`
 `;
 
 const ResourceTitle = styled(Stack)`
-  margin: 10px 0;
+  margin: 5px 0;
```
Comment (tdurnford) [inline extensions/azurePublish/src/components/ChooseProvisionAction.tsx:65]: We should try to use multiples of 4.
```suggestion
  margin: 4px 0;
```

File: extensions/azurePublish/src/components/ResourceGroupPicker.tsx [hunk 4]
```diff
@@ -12,10 +12,11 @@ import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
 
 import { useDebounce } from './useDebounce';
 
-const stackStyles = { root: { marginBottom: '8px' } };
-const dropdownStyles = { root: { marginBottom: '8px' }, dropdown: { width: '100%' } };
+const stackStyles = { root: { marginBottom: '6px' } };
+
```
Comment (tdurnford) [inline extensions/azurePublish/src/components/ResourceGroupPicker.tsx:16]: nit: unnecessary newline

File: extensions/azurePublish/src/components/azureProvisionDialog.tsx [hunk 5]
```diff
@@ -86,6 +86,8 @@ const iconStyle = (required) => {
   };
 };
 
+const resourceFieldStyles = { root: { paddingBottom: '6px', width: '75%' } };
```
Comment (tdurnford) [inline extensions/azurePublish/src/components/azureProvisionDialog.tsx:89]: We should try to use multiples of 4.
```suggestion
const resourceFieldStyles = { root: { paddingBottom: '8px', width: '75%' } };
```

**Review by tdurnford — CHANGES_REQUESTED** [review]

File: extensions/azurePublish/src/components/ChooseProvisionAction.tsx (hunk 1 above)
Comment (pavolum) [reply extensions/azurePublish/src/components/ChooseProvisionAction.tsx:142]: Handled

**Review by tdurnford — APPROVED** [review]
Beautiful

**Review by cwhitten — APPROVED** [review]

**Pull request merged**